app/lib/ui/main_screen.dart

```diff
@@ -36,41 +40,75 @@ class _MainScreenState extends State<MainScreen> {
       ),
     );
 
-    return Scaffold(
-        appBar: AppBar(
-          title: Text(_MainScreenConfig.mainMenuItems[_selectedIndex].label),
-        ),
-        bottomNavigationBar:
-            MediaQuery.of(context).size.width < mobileUiMaxScreenWidth
-                ? BottomNavigationBar(
-                    type: BottomNavigationBarType.fixed,
-                    items: _bottomNavigationBarItems,
-                    currentIndex: _selectedIndex,
-                    onTap: (value) {
-                      setState(() {
-                        _selectedIndex = value;
-                      });
-                    },
-                  )
-                : null,
-        body: Row(
-          mainAxisSize: MainAxisSize.max,
-          children: [
-            if (MediaQuery.of(context).size.width >= mobileUiMaxScreenWidth)
-              NavigationRail(
-                extended: MediaQuery.of(context).size.width >=
-                    extendedNavigationRailMinScreenWidth,
-                destinations: _navigationRailDestinations,
-                selectedIndex: _selectedIndex,
-                onDestinationSelected: (value) {
-                  setState(() {
-                    _selectedIndex = value;
-                  });
-                },
-              ),
-            Expanded(child: mainArea),
-          ],
-        ));
+    return Consumer<ModelTheme>(builder: (context, ModelTheme themeNotifier, child)
+    {
+      return Scaffold(
+          appBar: AppBar(
+            title: Text(_MainScreenConfig.mainMenuItems[_selectedIndex].label),
+            actions: [
+              DropdownButton<String>(
+                  value: chosenTheme,
+                  items: themeList.map<DropdownMenuItem<String>>((String value) {
+                    return DropdownMenuItem<String>(
```

Comment:
put it in a dedicated widget ThemeSelector for example

Reply:
Yep indeed way better if we do that. Let me tackle it in a separate branch + PR.